Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift

```diff
@@ -0,0 +1,551 @@
+import CTensorFlow
+
+/// The `TF_Tensor *` type.
+typealias CTFTensor = OpaquePointer
```

Comment:
Could this be `fileprivate`?

Reply:
Actually, this is an artifact of some earlier version. I removed this altogether.